_includes/js/objects.md

```diff
@@ -199,6 +199,29 @@ teamMember.save(null, { cascadeSave: false });
 
 ```
 
+### Cloud Code context
+
+You may pass a `context` that is accessible in Cloud Code `beforeSave` and `afterSave` triggers for that `Parse.Object`. This is useful if you want to condition certain operations in Cloud Code triggers on ephemeral information that should not be saved with the `Parse.Object` in the database. The context is ephemeral in the sense that it vanishes after the Cloud Code triggers for that particular `Parse.Object` have executed. For example:
```

Comment:
```suggestion
You may pass a `context` dictionary that is accessible in Cloud Code `beforeSave` and `afterSave` triggers for that `Parse.Object`. This is useful if you want to condition certain operations in Cloud Code triggers on ephemeral information that should not be saved with the `Parse.Object` in the database. The context is ephemeral in the sense that it vanishes after the Cloud Code triggers for that particular `Parse.Object` have executed. For example:
```
Or ‘`context` object’ if you prefer?

Reply:
The context is currently in `afterSave Triggers` > `Using Request Context`.

Should I move it into its own main section, since it belongs to both triggers and beyond?

Reply:
That sounds like a plan 👌

Reply:
Check it out, looks much cleaner now.

- added headings per trigger group
- moved context of save triggers into own section

![image](https://user-images.githubusercontent.com/5673677/79671026-661ef000-81c7-11ea-8ecf-02ff9a11e380.png)

Reply:
That’s a lot better!